tcs-service/src/main/resources/db/migration/schema/V4.27__add_fundingsubtype_to_postfunding.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `PostFunding` ADD `fundingSubType` varchar(36);
```

Comment:
I'd prefer "ADD COLUMN" and it would be consistent with the other existing sql scripts.